nestkernel/event_delivery_manager.cpp

```diff
@@ -679,12 +673,8 @@ EventDeliveryManager::gather_target_data( const thread tid )
     kernel().connection_manager.clean_source_table( tid );
 
 #ifdef TIMER_DETAILED
-#pragma omp barrier
```

Comment:
I think meanwhile that it is more logical at this point to put sw_communicate_target_data.start()/stop() into the "omp single" region.

Reply:
Otherwise I am happy with the changes, thank you! (@hakonsbm @jarsi @terhorstd @suku248)